tests/test_graph_api.py

```diff
@@ -684,6 +685,7 @@ def test_get_unicode_url():
     mock_request.return_value.status_code = 200
 
     response = graph.get('https://www.facebook.com/christophernewportuniversity‎')
+    del response['headers']
```

Comment:
I think you should be able to just set ´mock_request.return_value.headers`, which would allow you to assert that `response['headers]` contains what you expect.